packages/editor/src/store/test/reducer.js

```diff
@@ -2422,15 +2422,13 @@ describe( 'state', () => {
 						raw: 'The Excerpt',
 					},
 					status: 'draft',
-					preview_link: 'https://wordpress.org/?p=1&preview=true',
```

Comment:
It looks like this missing `preview=true` part in the e2e is expected.

Reply:
It should be fixed